marshaller.go

```diff
@@ -116,6 +116,14 @@ func (a *AuditMarshaller) completeMessage(seq int) {
 		//TODO: attempted to complete a missing message, log?
 		return
 	}
+	for _, m := range msg.Msgs {
+		switch m.Type {
+		case 1306:
+			// delay the mapping
+			time.Sleep(time.Millisecond * 100)
+			msg.mapDns(m)
```

Comment:
If map dns fails then you need to exit and let the dnstap input flush this event